docs/source/conf.py

```diff
@@ -103,6 +103,14 @@
 # TODO: verify this works as expected
 release = 'master'
 
+# Customized html_title here. 
+# Default is " ".join(project, release, "documentation") if not set
+if RELEASE:
+    html_title = " ".join((project, torch.__version__, "documentation"))    
```

Comment:
Could you comment with a screenshot of the documentation as a quick test?

Reply:
This is in header. It changed to 
<title>PyTorch documentation — PyTorch 1.6.0 documentation</title>
now

Reply:
I was asking for a screenshot of the actual webpage that says that, as a sanity check that this code gets threaded all the way through, but I believe that it works :)

Reply:
I see. Can you review this first https://github.com/pytorch/pytorch/pull/38119.
So I can empty my github.io to put another demo there.

Reply:
Sounds good, I'll go review that first. If you have a local version of the build, you can open the html file directly on your computer and then take a screenshot of that.